torch/cuda/streams.py

```diff
@@ -112,6 +112,29 @@ def __repr__(self):
                 .format(self.device, self.cuda_stream))
 
 
+class ExternalStream(Stream):
+    r"""Wrapper around a externally allocated CUDA stream.
+
+    A CUDA stream is a linear sequence of execution that belongs to a specific
```

Comment:
Let's avoid circular references "independent from other streams"
A CUDA stream is a sequence of operations that execute in order on a specific device.

Reply:
I just deleted that comment and replaced the entire docstring by
```
r"""Wrapper around an externally allocated CUDA stream.

    This class is used to wrap streams allocated in other libraries in order
    to facilitate data exchange and multi-library interactions.

    This class doesn't manage the stream life-cycle, it is the user
    responsibility to keep the referenced stream alive while this class is
    being used.

    Args:
...
```

How do you think?
Thanks!